### ISSUE TITLE
Can't install on ubuntu 21.04

### ISSUE THREAD

**Issue by ghost** [rethinkdb/rethinkdb#6982]
I am trying to install RethinkDB on ubuntu 21.04, however I am getting an error. 

Console:
```bash
ubuntu@vps-ad4ce1e1:~/database$ source /etc/lsb-release && echo "deb https://dow                                            nload.rethinkdb.com/repository/ubuntu-$DISTRIB_CODENAME $DISTRIB_CODENAME main"                                             | sudo tee /etc/apt/sources.list.d/rethinkdb.list
wget -qO- https://download.rethinkdb.com/repository/raw/pubkey.gpg | sudo apt-ke                                            y add -
sudo apt-get update
sudo apt-get install rethinkdb
deb https://download.rethinkdb.com/repository/ubuntu-hirsute hirsute main
Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (s                                            ee apt-key(8)).
OK
Hit:1 http://security.ubuntu.com/ubuntu hirsute-security InRelease
Hit:2 http://nova.clouds.archive.ubuntu.com/ubuntu hirsute InRelease
Get:3 http://nova.clouds.archive.ubuntu.com/ubuntu hirsute-updates InRelease [10                                            9 kB]
Hit:4 https://deb.nodesource.com/node_16.x hirsute InRelease
Ign:5 https://download.rethinkdb.com/repository/ubuntu-hirsute hirsute InRelease
Get:6 http://nova.clouds.archive.ubuntu.com/ubuntu hirsute-backports InRelease [                                            101 kB]
Err:7 https://download.rethinkdb.com/repository/ubuntu-hirsute hirsute Release
  404  Repository not found [IP: 167.172.11.96 443]
Reading package lists... Done
E: The repository 'https://download.rethinkdb.com/repository/ubuntu-hirsute hirs                                            ute Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disa                                            bled by default.
N: See apt-secure(8) manpage for repository creation and user configuration deta                                            ils.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package rethinkdb is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'rethinkdb' has no installation candidate
ubuntu@vps-ad4ce1e1:~/database$
```

What I'm doing wrong?

**Comment by mStirner** [comment]
Have you tried to compile it by yourself? 
Seems like there is no pre-compiled package for your OS.

**Comment by St0rmdev** [comment]
i downgraded os to ubuntu 20.04, manual installation gives me a error

**Comment by mStirner** [comment]
Can you post the error message?

**Comment by yaneony** [comment]
Possible dupe? https://github.com/rethinkdb/rethinkdb/issues/6959

**Comment by St0rmdev** [comment]
> Can you post the error message?

unfortunately i dont have error message, sorry for not responding

**Comment by srh** [comment]
This has been fixed in v2.4.x branch -- building and package generation on latest Ubuntus is working -- and release v2.4.2 will have a 21.04 and 21.10 package.  See #6975 for updates.
